R/plot.R

```diff
@@ -429,16 +429,20 @@ contour.stars = function(x, ...) {
 }
 
 #' reduce dimension to rgb (alpha) hex values
-#' 
+#'
 #' @export
 #' @param x object of class \code{stars}
 #' @param dimension dimension name or number to reduce
 #' @param use_alpha logical; if TRUE, the fourth band will be used as alpha values
 #' @param maxColorValue integer; maximum value for colors
-#' @param stretch logical; if \code{TRUE}, each band is stretched to 0 ... \code{maxColorValue}
-#' @param probs probability values for quantiles used for stretching
+#' @param stretch logical or character; if \code{TRUE}, each band is stretched to 0 ... \code{maxColorValue}
+#' by "percent clip"; if character \code{"percent"} is required \code{probs}; if
+#' character \code{"histogram"} a "histogram equalization" is performed. Other character
+#' values are ignored, insted method used is "percent".
```

Comment:
insted -> instead; could you please try to improve the explanation of these options, I read it a couple of times and still don't understand it.

Reply:
Yes, It is weird, sorry. I will improve in a new PR

Reply:
Sorry for the noise in the PRs A friend of mine recommended not open a new PR. So I closed the new one and reopened ir.